data/json/recipes/recipe_food.json

```
@@ -1073,6 +1146,7 @@
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
     "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
+    "proficiencies": [ { "proficiency": "prof_food_prep" } ],
```

```suggestion
    "proficiencies": [ { "proficiency": "prof_food_prep" }, { "proficiency": "prof_knife_skills" } ],
```
Catfood knife skills